src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,69 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String stringFromFile = readFile(fromFileName);
+        String[][] separatedData = separateData(stringFromFile);
+        int[] results = calculateData(separatedData);
+        String report = createReport(results);
+        File file = new File(toFileName);
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(file))) {
+            writer.write(report);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write data to file", e);
+        }
```

Comment:
```suggestion
        File file = new File(toFileName);
        try (BufferedWriter writer = new BufferedWriter(new FileWriter(file))) {
            writer.write(report);
        } catch (IOException e) {
            throw new RuntimeException("Can't write data to file " + toFileName, e);
        }
```

lets also move logic of file writing to separate method. And try to make exception messages more informative